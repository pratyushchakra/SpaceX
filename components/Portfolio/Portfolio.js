import Filters from '../Filters/Filters';
import styles from './Portfolio.style';
import Pods from '../Pods/Pods'
const Portfolio = () => {
    return (
        <div className="page-layout">
            <div className="filters-container">
                <Filters />
            </div>
            <div className="pods-container">
                <Pods />
            </div>
            <style jsx>{styles}</style>
        </div>
    )
}

export default Portfolio