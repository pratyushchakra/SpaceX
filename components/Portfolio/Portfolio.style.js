import css from 'styled-jsx/css';

const PortfolioStyles = css`
.page-layout {
    display:flex;
}
.filters-container{
    width: 16%;
    background: #fff;
    min-width: 170px;
}
.pods-container {
    width: 84%;
    display: grid;
    margin: 0 20px;
}

@media only screen and (max-width: 700px) {
.pods-container {
    grid-template-columns: 100%;
}
.page-layout {
    flex-direction :column;
    text-align: center;
}
.filters-container{
    width: 90%;
    margin: 0 auto;
}
.pods-container {
    width: 90%;
    margin: 0 auto;
}
}
@media only screen and (min-width: 700px) {
.pods-container {
    grid-template-columns: 50% 50%;
}
}
@media only screen and (min-width: 1024px) {
.pods-container {
    grid-template-columns: 25% 25% 25% 25%;
}
}
`
export default PortfolioStyles;