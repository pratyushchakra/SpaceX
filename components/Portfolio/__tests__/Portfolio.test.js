import Enzyme, { shallow, mount } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';
import Portfolio from '../Portfolio';
Enzyme.configure({ adapter: new Adapter() })

describe('Portfolio Component', () => {
    it('it should render as expected', () => {
        const wrapper = shallow(
            <Portfolio />
        );
        expect(wrapper.debug()).toMatchSnapshot()
    });
})