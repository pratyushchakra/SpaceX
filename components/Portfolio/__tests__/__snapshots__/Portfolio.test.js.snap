// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Portfolio Component it should render as expected 1`] = `
"<div className=\\"jsx-1419685100 page-layout\\">
  <div className=\\"jsx-1419685100 filters-container\\">
    <Filters />
  </div>
  <div className=\\"jsx-1419685100 pods-container\\">
    <Pods />
  </div>
  <JSXStyle id=\\"1419685100\\">
    .
    p
    a
    g
    e
    -
    l
    a
    y
    o
    u
    t
    .
    j
    s
    x
    -
    1
    4
    1
    9
    6
    8
    5
    1
    0
    0
    {
    d
    i
    s
    p
    l
    a
    y
    :
    -
    w
    e
    b
    k
    i
    t
    -
    b
    o
    x
    ;
    d
    i
    s
    p
    l
    a
    y
    :
    -
    w
    e
    b
    k
    i
    t
    -
    f
    l
    e
    x
    ;
    d
    i
    s
    p
    l
    a
    y
    :
    -
    m
    s
    -
    f
    l
    e
    x
    b
    o
    x
    ;
    d
    i
    s
    p
    l
    a
    y
    :
    f
    l
    e
    x
    ;
    }
    .
    f
    i
    l
    t
    e
    r
    s
    -
    c
    o
    n
    t
    a
    i
    n
    e
    r
    .
    j
    s
    x
    -
    1
    4
    1
    9
    6
    8
    5
    1
    0
    0
    {
    w
    i
    d
    t
    h
    :
    1
    6
    %
    ;
    b
    a
    c
    k
    g
    r
    o
    u
    n
    d
    :
    #
    f
    f
    f
    ;
    m
    i
    n
    -
    w
    i
    d
    t
    h
    :
    1
    7
    0
    p
    x
    ;
    }
    .
    p
    o
    d
    s
    -
    c
    o
    n
    t
    a
    i
    n
    e
    r
    .
    j
    s
    x
    -
    1
    4
    1
    9
    6
    8
    5
    1
    0
    0
    {
    w
    i
    d
    t
    h
    :
    8
    4
    %
    ;
    d
    i
    s
    p
    l
    a
    y
    :
    g
    r
    i
    d
    ;
    m
    a
    r
    g
    i
    n
    :
    0
     
    2
    0
    p
    x
    ;
    }
    @
    m
    e
    d
    i
    a
     
    o
    n
    l
    y
     
    s
    c
    r
    e
    e
    n
     
    a
    n
    d
     
    (
    m
    a
    x
    -
    w
    i
    d
    t
    h
    :
    7
    0
    0
    p
    x
    )
    {
    .
    p
    o
    d
    s
    -
    c
    o
    n
    t
    a
    i
    n
    e
    r
    .
    j
    s
    x
    -
    1
    4
    1
    9
    6
    8
    5
    1
    0
    0
    {
    g
    r
    i
    d
    -
    t
    e
    m
    p
    l
    a
    t
    e
    -
    c
    o
    l
    u
    m
    n
    s
    :
    1
    0
    0
    %
    ;
    }
    .
    p
    a
    g
    e
    -
    l
    a
    y
    o
    u
    t
    .
    j
    s
    x
    -
    1
    4
    1
    9
    6
    8
    5
    1
    0
    0
    {
    -
    w
    e
    b
    k
    i
    t
    -
    f
    l
    e
    x
    -
    d
    i
    r
    e
    c
    t
    i
    o
    n
     
    :
    c
    o
    l
    u
    m
    n
    ;
    -
    m
    s
    -
    f
    l
    e
    x
    -
    d
    i
    r
    e
    c
    t
    i
    o
    n
     
    :
    c
    o
    l
    u
    m
    n
    ;
    f
    l
    e
    x
    -
    d
    i
    r
    e
    c
    t
    i
    o
    n
     
    :
    c
    o
    l
    u
    m
    n
    ;
    t
    e
    x
    t
    -
    a
    l
    i
    g
    n
    :
    c
    e
    n
    t
    e
    r
    ;
    }
    .
    f
    i
    l
    t
    e
    r
    s
    -
    c
    o
    n
    t
    a
    i
    n
    e
    r
    .
    j
    s
    x
    -
    1
    4
    1
    9
    6
    8
    5
    1
    0
    0
    {
    w
    i
    d
    t
    h
    :
    9
    0
    %
    ;
    m
    a
    r
    g
    i
    n
    :
    0
     
    a
    u
    t
    o
    ;
    }
    .
    p
    o
    d
    s
    -
    c
    o
    n
    t
    a
    i
    n
    e
    r
    .
    j
    s
    x
    -
    1
    4
    1
    9
    6
    8
    5
    1
    0
    0
    {
    w
    i
    d
    t
    h
    :
    9
    0
    %
    ;
    m
    a
    r
    g
    i
    n
    :
    0
     
    a
    u
    t
    o
    ;
    }
    }
    @
    m
    e
    d
    i
    a
     
    o
    n
    l
    y
     
    s
    c
    r
    e
    e
    n
     
    a
    n
    d
     
    (
    m
    i
    n
    -
    w
    i
    d
    t
    h
    :
    7
    0
    0
    p
    x
    )
    {
    .
    p
    o
    d
    s
    -
    c
    o
    n
    t
    a
    i
    n
    e
    r
    .
    j
    s
    x
    -
    1
    4
    1
    9
    6
    8
    5
    1
    0
    0
    {
    g
    r
    i
    d
    -
    t
    e
    m
    p
    l
    a
    t
    e
    -
    c
    o
    l
    u
    m
    n
    s
    :
    5
    0
    %
     
    5
    0
    %
    ;
    }
    }
    @
    m
    e
    d
    i
    a
     
    o
    n
    l
    y
     
    s
    c
    r
    e
    e
    n
     
    a
    n
    d
     
    (
    m
    i
    n
    -
    w
    i
    d
    t
    h
    :
    1
    0
    2
    4
    p
    x
    )
    {
    .
    p
    o
    d
    s
    -
    c
    o
    n
    t
    a
    i
    n
    e
    r
    .
    j
    s
    x
    -
    1
    4
    1
    9
    6
    8
    5
    1
    0
    0
    {
    g
    r
    i
    d
    -
    t
    e
    m
    p
    l
    a
    t
    e
    -
    c
    o
    l
    u
    m
    n
    s
    :
    2
    5
    %
     
    2
    5
    %
     
    2
    5
    %
     
    2
    5
    %
    ;
    }
    }
  </JSXStyle>
</div>"
`;
